lib/dndfonts.sty

```diff
@@ -18,3 +19,5 @@
 
 \newcommand{\dnd@FooterFont}{\normalfont\scshape}
 \newcommand{\dnd@PageNumberFont}{\normalfont}
+
+\newcommand\dnd@DropCapFont{\ArtNouvcfamily}
```

Comment:
ArtNouveauc is a nice font, but it is problematic as the default because of all the parts of different letters that extend to the right and downwards. I would propose using a different font, one that has the letters fit more within the box. http://www.tug.dk/FontCatalogue/otherfonts.html

RoyalInitialen is a possiblity, although the styling may not work for some, and most of the fonts are problematic in that regard: there is not really one that works for everybody's preference.

```
\input RoyalIn.fd
\newcommand*\RoyalInfamily{\usefont{U}{RoyalIn}{xl}{n}}
\newcommand\dnd@DropCapFont{\RoyalInfamily}
\renewcommand{\LettrineFontHook}{\dnd@DropCapFont}
```